UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetForLargeFile.cs

```diff
@@ -61,23 +60,37 @@ public void WhenICallExchangeSetApiWithMultipleProductIdentifiers_ThenAMediaTxtF
 
         [Test]
         [Category("QCOnlyTest")]
-        public void WhenICallExchangeSetApiWithMultipleProductIdentifiers_ThenAnINFOFolderIsGenerated()
+        public void WhenICallExchangeSetApiWithMultipleProductIdentifiers_ThenAnINFOFolderWithFilesIsGenerated()
         {
+            string[] infoFolderFiles = { "AVCS-User-Guide.pdf", "ENC TandP NM status.pdf", "End-User-Licence-Agreement-for-ADMIRALTY-digital-data-services.pdf", "Important Information for AVCS users.pdf" };
             foreach (string folderPath in DownloadedFolderPath)
```

Comment:
is there one folder in the INFO or are we creating multiple

Reply:
As per our discussion no need to fix this.

Thanks,
Mayuresh Gawde